lib/scrivener/html.ex

```diff
@@ -152,13 +152,12 @@ defmodule Scrivener.HTML do
             classes = ["active"]
           end
           params_with_page = Dict.merge(url_params, page: page_number)
-          content_tag :li do
+          content_tag :li, class: Enum.join(classes, " ") do
             to = apply(path, args ++ [params_with_page])
-            class = Enum.join(classes, " ")
             if to do
-              link "#{text}", to: apply(path, args ++ [params_with_page]), class: class
+              link "#{text}", to: apply(path, args ++ [params_with_page])
```

Comment:
Just noticed that we are applying the path twice! Since you're in here, would you mind replacing this with the variable `to`?